mm/kfence/report.c

```diff
@@ -177,6 +178,7 @@ void kfence_report_error(unsigned long address, const struct kfence_metadata *me
 	 */
 	lockdep_off();
 
+	trace_error_report_start("kfence", address);
```

Comment:
Do we want the report with or without the '==============='.

I think it should be without, because it doesn't add much value in the file and just eats space from the buffer.

Reply:
Yes, very good point. We also don't have standard separator length across tools anyway.